search-app/src/cmr/search/validators/all_granule_validation.clj

```diff
@@ -55,3 +55,16 @@
            "collections such as provider, concept_id, short_name, or entry_title.")
       (:offset query) (all-granules-page-depth-limit))]))
 
+(defn no-all-granules-with-scroll
+  "Validates that the query is not an all granules query if it is a scroll query."
+  [query]
+  (when (and (all-granules-query? query) 
```

Comment:
Nit: We should put the `(:scroll query)` check first since it will short-circuit faster.